guides/DigitalOcean Tutorial.md

```diff
@@ -6,41 +6,46 @@ DigitalOcean is a cloud infrastructure provider focused on simplifying web infra
-Next, we need to choose the specs for our server. In this tutorial, we'll be using the most basic tier of a Standard `Droplet`, which offers a single CPU with 512MB RAM, 20GB SSD and 1000 GB transfer at $5 per month. Generally, it's more than enough for running personal applications. You may also run several services in a single Droplet.
+Next, we need to choose the specs for our server. In this tutorial, we'll be using the most basic tier of a Standard `Droplet`, which offers a single Regular Intel CPU with 1GB RAM, 25GB SSD and 1000 GB transfer at $5 per month. Generally, it's more than enough for running personal applications. You may also run several services in a single Droplet.
 
 ### Choosing a datacenter region
 
 Generally, choosing a region that's _closest to your users_ will make your service deliver faster. If your users are primarily in the United States, you could choose a United States-based Droplet.
 
 ### Other configurations
 
-In our example, we do not need to add any other services such as block storage or private network. So we will ignore these settings to keep our setup simple and cheap. You may choose to use `SSH` key or you can just leave it unchecked and a password will be generated for you. If you choose to use SSH key, then each time you want to login to the server, you will need to provide the key. Since you have to have access to the SSH key whenever you log in, it can be more secure or more troublesome depends on scenarios. At last, you may change the name of your `Droplet` to something you like and then click `Create` to create and launch your `Droplet`.
+In our example, we do not need to add any other services such as block storage or private network. So we will ignore these settings to keep our setup simple and cheap. You may choose to use `SSH` key and follow the steps recommended by DigitalOcean or create a password. If you choose to use SSH key, then each time you want to login to the server from outside DigitalOcean's site, you will need to provide the key. For example in windows using [GitBash terminal](https://git-scm.com/downloads) you need to add the `-i` option and pass the full path for the ssh-key file:
 
-## Connecting to our droplet
+```bash
+ssh root@insert-droplet-ip -i /c/Users/insert-username/.ssh/insert-ssh-key-filename
+```
 
-![DigitalOcean Access Console](assets/DigitalOcean/access_console.png)
+Be aware that this command will only work for the root user, if you want to add an ssh key for another user you can see the details in [DigitalOcean's docs](https://docs.digitalocean.com/products/droplets/how-to/add-ssh-keys/).
 
-Once you've created your `Droplet`, you will receive an e-mail containing your login (`root`) and initial password for it. Now click the "Access Console" option in the dropdown menu associated with your `Droplet` as shown in above image.
+Since you have to have access to the SSH key whenever you log in, it can be more secure or more troublesome depending on the scenario.  
```

Comment:
I'm all for using SSH keys, but in the course I believe we use password (because for most students, the SSH key may be very confusing). Should we keep a section here that uses password, and add a section that uses SSH keys? That way they can use one or the other but the content covered in the videos is still here in the guide.

Reply:
Updated